extensions/wikia/WikiaNewFiles/WikiaNewFiles.class.php

```diff
@@ -20,4 +65,147 @@ function execute( $par ) {
 	public function getDescription() {
 		return $this->msg( 'wikianewfiles-title' )->text();
 	}
+
+	private function getLinkedFiles( $image ) {
+		global $wgMemc;
+		$anchorLength = 60;
+
+		wfProfileIn( __METHOD__ );
+		$cacheKey = wfMemcKey( __METHOD__, md5( $image->img_name ) );
```

Comment:
Lets take this opportunity to pull these out.  We've got `influxdb's app_agg_xhprof_funcs` for this so these just add clutter, execution time and confusion.